src/user/using-the-capture-app.md

```diff
@@ -872,6 +872,21 @@ On the enrollment dashboard, you can view the Person Profile widget.
 
 The Person profile widget is used for viewing key attributes but not for editing. In order to edit the person profile, you must select the `Edit` button to open an edit window. 
 
+### Indicator widget
+
+On the enrollment dashboard, you can view the Indicator widget.
+
+![](resources/images/capture_app/enrollment-dash-indicator-widget-1.png)
+
+The indicator widget displays program indicators that are configured as "Display in form" or through program rules (Displaying text or key/value pairs in "Program indicator widget"). 
```

Comment:
I suggest rewording to simplify and remove too much DHIS2 data model:
`On the enrollment dashboard, the indicator widget displays indicator text and values output related to the current dashboard.`

I don't think the user documentation needs to refer to how program indicators are set up - this seems more relevant to implementer/configuration documentation. What do you think @Bekkalizer / @karolinelien?

Reply:
To me the user documentation is a mix of user and configuration documentation - for example the chapter on "configuring the system", specifically on program rules here: https://docs.dhis2.org/en/use/user-guides/dhis-core-version-236/configuring-the-system/programs.html#configure_program_rule - maybe we should specify it more there instead (Just says "display widget" - could say this refers to Indicator widget in Capture app) - I just want to make sure the user/configurer can read about what to do to actually get the indicators to display in the widget _somewhere_ in the docs :)

Reply:
I think the current level of detail is fine for the user documentation related to DHIS2-11212.